src/static/bitbucket/components/App.tsx

```diff
@@ -1,7 +1,13 @@
 import * as React from 'react';
 import { proxyRequest } from '../utils/RequestProxy';
 
-export type AppState = {
+type BannerMessage = {
+  messageExists: boolean;
+  message: string | null;
```

Comment:
if message is null it will be falsy so ```if (!message) {
        res.status(400).json({ error: 'Cannot send an empty message' });
      } ```will return 400 - do we want that ? Maybe we just do message: string

Reply:
I see below that you pass a null value for message